src/test/operate.test.js

```diff
@@ -0,0 +1,9 @@
+import operate from '../logic/operate';
+
+describe('test the logic of the application', () => {
+  it('take two numbers and return the operation of them', () => {
+    expect(operate(2, 5, '+')).toBe('7');
+    expect(operate(2, 5, '-')).toBe('-3');
+    expect(operate(2, 5, 'x')).toBe('10');
+  });
+});
```

Comment:
Kindly test all arithmetic operations. You are missing tests for division (÷) and modulus (%).

- Please make sure that each test has its own description that explains the operation being tested, in your case you can have something like:

```
  it('Test for addition operation', () => {
    expect(operate(2, 5, '+')).toBe('7');
  });

  it('Test for subtraction operation', () => {
    expect(operate(2, 5, '-')).toBe('-3');
  });

... and so forth.
```

Another example:
```
describe('testing operations', () => {
  test('test addition', () => {
    const result = operate(5, 9, '+');
    expect(result).toBe('14');
  });

  test('test subtraction', () => {
    const result = operate(5, 9, '-');
    expect(result).toBe('-4');
  });

  test('test multiplication', () => {
    const result = operate(5, 9, 'x');
    expect(result).toBe('45');
  });

  test('test division', () => {
    const result = operate(5, 9, '÷');
    expect(result).toBe('0.55555555555555555556');
  });

  test('test modulus', () => {
    const result = operate(5, 9, '%');
    expect(result).toBe('5');
  });
});
```